test/models/concerns/foreman/observable_model_test.rb

```diff
@@ -2,8 +2,8 @@
 
 class ObservableModelTest < ActiveSupport::TestCase
   describe '#set_hook' do
-    let(:model) { klass.create(name: 'My Model') }
-    let(:klass) do
+    let(:model) { model_class.create(name: 'My Model') }
+    let(:model_class) do
```

Comment:
TIL: If you create `klass` helper like that, it breaks the output to something like:

```
#<Class:0x0000000010500928>#test_0001_event subscription hooks are defined in the the Model class = 0.81 s = .
#<Class:0x000000001056c0d8>#test_0003_notify with default payload = 0.03 s = .
#<Class:0x00000000106bd5e0>#test_0006_notify with if condition = 0.02 s = .
#<Class:0x00000000107cc0f8>#test_0004_notify with custom payload = 0.02 s = .
#<Class:0x00000000108dbb10>#test_0005_notify with block payload = 0.02 s = .
#<Class:0x00000000109cf3a0>#test_0002_notify with event context = 0.02 s = .
#<Class:0x0000000010acdef0>#test_0001_event is sent when model is created = 0.01 s = .
#<Class:0x0000000010bb1358>#test_0001_event is sent when model is destroyed = 0.02 s = .
#<Class:0x0000000010ca4878>#test_0001_event is sent when model is updated = 0.23 s = .
#<Class:0x00000000072eff20>#test_0001_hooks are defined = 0.01 s = .
```